tsl/test/sql/compression.sql

```diff
@@ -757,3 +757,68 @@ SET enable_indexscan to off;
 SELECT time, const, numeric,first, avg1, avg2 FROM tidrangescan_expr ORDER BY time LIMIT 5;
 RESET timescaledb.enable_chunk_append;
 RESET enable_indexscan;
+
+
+
+-- Improve the number of parallel workers for decompression
+
+-- Test that a parallel plan is generated
+-- with different number of parallel workers
+
+
+
+CREATE TABLE f_sensor_data(
+      time timestamptz not null,
+      sensor_id integer not null,
+      cpu double precision null,
+      temperature double precision null
+    );
+
+SELECT FROM create_hypertable('f_sensor_data','time');
+SELECT set_chunk_time_interval('f_sensor_data', INTERVAL '1 year');
+
+SELECT * FROM _timescaledb_internal.create_chunk('f_sensor_data',' {"time": [181900977000000, 515024000000000]}');
+
+INSERT INTO f_sensor_data
+SELECT
+    time AS time,
+    sensor_id,
+    100.0,
+    36.6
+FROM
+    generate_series('1980-01-01 00:00'::timestamp, '1980-02-28 12:00', INTERVAL '1 day') AS g1(time),
+    generate_series(1, 100, 1 ) AS g2(sensor_id)
+ORDER BY
+    time;
+
+ALTER TABLE f_sensor_data SET (timescaledb.compress, timescaledb.compress_segmentby='sensor_id' ,timescaledb.compress_orderby = 'time DESC');
+
+SELECT add_compression_policy('f_sensor_data','1 minute'::INTERVAL);
+SELECT compress_chunk(i) FROM show_chunks('f_sensor_data') i;
+
+-- Encourage use of parallel plans
+SET parallel_setup_cost = 0;
+SET parallel_tuple_cost = 0;
+
+SET min_parallel_table_scan_size TO '1';
+
+\set explain 'EXPLAIN (VERBOSE, COSTS OFF)'
```

Comment:
just a thought, as I have not experimented to find the appropriate pattern matching: maybe adding an appropriate sed pattern in `test/runner.sh` would help to remove the test flakiness that ANALYZE introduces and thus allow ANALYZE to show that workers have been planned for the following queries?

Reply:
I have removed ANALYZE 
because the rows of each parallel worker always changing
and the tests where failing.